metrics_sdk/test/integration/in_memory_metric_pull_exporter_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::SDK do
+  describe '#configure' do
+    before { reset_metrics_sdk }
+
+    it 'emits metrics' do
+      OpenTelemetry::SDK.configure
+
+      metric_exporter = OpenTelemetry::SDK::Metrics::Export::InMemoryMetricPullExporter.new
+      OpenTelemetry.meter_provider.add_metric_reader(metric_exporter)
+
+      meter = OpenTelemetry.meter_provider.meter('test')
+      instrument = meter.create_counter('b_counter')
+
+      instrument.add(1)
+      instrument.add(2, attributes: { 'a' => 'b' })
+      instrument.add(2, attributes: { 'a' => 'b' })
+      instrument.add(3, attributes: { 'b' => 'c' })
+      instrument.add(4, attributes: { 'd' => 'e' })
```

Comment:
Do we want any expectations here about the metric streams?

Reply:
Added some assertions!